src/main/core/support/configuration.rs

```diff
@@ -237,7 +237,7 @@ pub struct ExperimentalOptions {
     #[clap(about = EXP_HELP.get("use_shim_syscall_handler").unwrap())]
     use_shim_syscall_handler: Option<bool>,
 
-    /// Use CPU pinning
+    /// Pin each thread and any processes it executes to the same CPU to improve cache affinity
```

Comment:
Maybe use "logical CPU core" to specify that you don't mean the same CPU package.